specs-go/v1/manifest.go

```diff
@@ -36,3 +36,16 @@ type Manifest struct {
 	// Annotations contains arbitrary metadata for the image manifest.
 	Annotations map[string]string `json:"annotations,omitempty"`
 }
+
+// ScratchDigestSHA256 is the digest of a blob with content of `{}` (size of 2)
+const ScratchDigestSHA256 = `sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a`
+
+// ScratchDescriptor is a SCRATCH descriptor for Manifest's `config` when that manifest does not have a config to provide.
+func ScratchDescriptor() Descriptor {
+	return Descriptor{
+		MediaType: MediaTypeImageScratch,
+		Digest:    ScratchDigestSHA256,
+		Size:      2,
+		Data:      []byte(`{}`),
```

Comment:
If we keep this section, this value of Data is invalid. It would be base64 encoded.